content/applications/essentials/export_import_data.rst

```diff
@@ -2,166 +2,245 @@
+      :align: center
+      :alt: Formatting options presented when a CVS file is imported in Odoo.
+
+Make necessary adjustments to the *Formatting* options, and ensure all columns in the
+:guilabel:`Odoo field` and :guilabel:`File Column` are free of errors. Finally, click
+:guilabel:`Import` to import the data.
+
+Adapt a template
+----------------
+
+Import templates are provided in the import tool of the most common data to import (contacts,
+products, bank statements, etc.). Open them with any spreadsheet software (*Microsoft Office*,
+*OpenOffice*, *Google Drive*, etc.).
+
+Once the template is downloaded, proceed to follow these steps:
 
-* Add, remove and sort columns to fit at best your data structure.
-* We advise to not remove the **ID** one (see why in the next section).
-* Set a unique ID to every single record by dragging down the ID sequencing.
+- Add, remove, and sort columns to best fit the data structure.
+- It is strongly advised to **not** remove the :guilabel:`External ID` (ID) column (see why in the
+  next section).
+- Set a unique ID to every record by dragging down the ID sequencing in the :guilabel:`External ID`
+  (ID) column.
 
 .. image:: export_import_data/dragdown.gif
    :align: center
+   :alt: An animation of the mouse dragging down the ID column, so each record has a unique ID.
 
-* When you add a new column, Odoo might not be able to map it automatically if its
-  label doesn't fit any field in Odoo. Don't worry! You can map
-  new columns manually when you test the import. Search the list for the
-  corresponding field.
+.. note::
+   When a new column is added, Odoo may not be able to map it automatically, if its label does not
+   fit any field within Odoo. However, new columns can be mapped manually when the import is tested.
+   Search the drop-down menu for the corresponding field.
+
+   .. image:: export_import_data/field_list.png
+      :align: center
+      :alt: Drop-down menu expanded in the initial import screen on Odoo.
+
+   Then, use this field's label in the import file to ensure future imports are successful.
+
+.. tip::
+   Another useful way to find out the proper column names to import is to export a sample file
+   using the fields that should be imported. This way, if there is not a sample import template,
+   the names are accurate.
+
+Import from another application
+-------------------------------
+
+The :guilabel:`External ID` (ID) is a unique identifier for the line item. Feel free to use one
+from previous software to facilitate the transition to Odoo.
 
-    .. image:: export_import_data/field_list.png
-       :align: center
+Setting an ID is not mandatory when importing, but it helps in many cases:
 
-  Then, use this field's label in your file in order to make it work
-  straight on the very next time.
+- Update imports: import the same file several times without creating duplicates.
+- Import relation fields (see :ref:`below <export_import_data/relation-fields>`).
```

Comment:
Can make this an inline descriptive reference, rather than the vague "below" label :)
```suggestion
- :ref:`Import relation fields <export_import_data/relation-fields>`).
```